### DISCUSSION TITLE
Piping roc-recv output

### DISCUSSION THREAD

**Discussion by nadigo** [roc-streaming/roc-toolkit#522] [Discuss]
What would be the best way to pipe roc-recv output to a pipe?

Sending with : 
`/usr/bin/roc-send -i file_example_WAV_10MG.wav --no-resampling -d wav   -s rtp+rs8m:${ROC_RECV_IP}:10001  -r rs8m:${ROC_RECV_IP}:10002`

to:
`usr/bin/roc-recv -vv -s rtp+rs8m::10001 -r rs8m::10002  --no-resampling  -d wav -o - > $STREAMSFOLDER/$PIPENAME`

doesn't work and I get choppy sound, any advice ?

**Comment by gavv** [comment]
I just tested this on fresh master (currently tagged as 0.2):

Sender:

```
roc-send -vv -s rtp+rs8m://${ROC_RECV_IP}:10001 -r rs8m://${ROC_RECV_IP}:10002 -i file:${PATH_TO_FILE}
```

Receiver + `paplay` (works with PulseAudio):

```
roc-recv -vv -s rtp+rs8m://0.0.0.0:10001 -r rs8m://0.0.0.0:10002 -o file://- --output-format wav | paplay
```

Receiver + `aplay` (works with PulseAudio and bare ALSA):

```
roc-recv -vv -s rtp+rs8m://0.0.0.0:10001 -r rs8m://0.0.0.0:10002 -o file://- --output-format wav | aplay
```

Receiver + `play` (comes with sox package; works with PulseAudio and bare ALSA):

```
roc-recv -vv -s rtp+rs8m://0.0.0.0:10001 -r rs8m://0.0.0.0:10002 -o file://- --output-format wav | play -
```

All three options work good for me. Please let me know if you have troubles with the above commands.
